internal/cli/alpha/manifestgen/helpers.go

```diff
@@ -38,6 +38,18 @@ func WriteManifestFiles(outputDir string, files map[string]string, override bool
 	return nil
 }
 
+//ManifestsExistsInOutputDir checks if manifests exists in dir
+func ManifestsExistsInOutputDir(files map[string]string, dir string) bool {
+	for manifestPath := range files {
+		manifestFilepath := strings.ReplaceAll(strings.TrimPrefix(manifestPath, "cap."), ".", string(os.PathSeparator)) + ".yaml"
```

Comment:
BTW there are two occurrences of the same line:
```
manifestFilepath := strings.ReplaceAll(strings.TrimPrefix(manifestPath, "cap."), ".", string(os.PathSeparator)) + ".yaml"
```

Let's have it in one place. Maybe we can even define it as a method on ManifestCollection? 🤔